### PR TITLE
another shared component fix

### LINKED ISSUE #170: Create eTransition pipeline funnel chart
See bnp dashboard
image

We need to create rest api endpoint in onboarding and also the chart component (take code from bnp and make a shared component in onboarding).

REST api
it should return dataset for the funnel
Identified:numberOfCampaignContacts total (all campaigns of tenant)
Contacted: numberOfContacts in tenant's campaigns any status but new, queued,generatingInvitation, invitationGenerated, sneding, bounced
Discussion: numberOfContacts in tenant's campaigns and status in (read, loaded, registered, serviceConfig, onboarded)
Won: numberOfContacts in tenant's campaigns and status = connected

### PR DESCRIPTION
**Pull request by theflexbird**
#170

### CODE REVIEW

**Comment by kwierchris** [comment]
This is already the second time you killed the service with these changes. Please test the stuff before you ask for a PR.

Client error:
Uncaught TypeError: Cannot read property 'default' of undefined
    at onboarding:20

**Comment by theflexbird** [comment]
On local it is woking as it should. It is an issue on the build side.

File: src/webpack/webpack.base.config.js [hunk 1]
```diff
@@ -3,10 +3,15 @@ const webpack = require('webpack');
 const Config = require('webpack-config').default;
 
 module.exports = new Config().merge({
-  entry: './src/client/index',
+  entry: {
+    onboarding: './src/client/index',
```
Comment (patrykkopycinski) [inline src/webpack/webpack.base.config.js]: it should be `main` instead of `onboarding` i believe

Comment (theflexbird) [inline src/webpack/webpack.base.config.js]: main is also not working

**Pull request closed without merging**